### PR TITLE
data:rghdrizzle

### PR DESCRIPTION
**Pull request by rghdrizzle**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4844"></a>

### CODE REVIEW

File: data/LuqmaanHakeem.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Luqmaan Hakeem",
+  "type": "personal",
+  "bio": "Just a typical devops guy (continously learning)",
+  "avatar":"https://twitter.com/rghdrizzle.png",
```
Comment (swagataroy30) [inline data/LuqmaanHakeem.json:5]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
In the recent update, we don't need to add avatar: anymore, it will auto-pick your GitHub profile picture. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

**Review by swagataroy30 — COMMENTED** [review]
you filename needs to match your Github username. you need to change it to 
`data/rghdrizzle.json`. Thank You!

**Comment by rghdrizzle** [comment]
If I have to update later in the future do I have to open a pr?

**Comment by kumarsonsoff3** [comment]
> If I have to update later in the future do I have to open a pr?

Exactly Luqmaan, You can always create a PR if you wanted to change anything in your Profile. 
You can follow this guide on `How to edit and add more to your profile`: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"></p>

**Pull request merged**